File: GroupiesMSTestsCSharp/UnitTest1.cs
namespace GroupiesMSTestsCSharp
{
    using Groupies;
    [TestClass]
    public class UnitTest1
    {
  
        [TestMethod]
        public void TestMethod1()
        {
            //var appController = new AppController();

            Assert.AreEqual("Es wurde keine Datei geladen", AppController.LoadFromJson("Test"));
        }
    }
}

File: GroupiesTestsVB/UnitTest1.vb
Imports Microsoft.VisualStudio.TestTools.UnitTesting
Imports Groupies

Namespace GroupiesTestsVB
    <TestClass>
    Public Class UnitTest1
        <TestMethod>
        Sub TestSub()
            Assert.AreEqual("Es wurde keine Datei geladen", AppController.LoadFromJson("Test"))
        End Sub
    End Class
End Namespace

